### PR TITLE
fix: Layout subNav, remove repo routes

### PR DESCRIPTION
**Pull request by whDongRui**

### CODE REVIEW

File: src/components/Layout/SideNav/index.jsx [hunk 1]
```diff
@@ -99,8 +99,7 @@ class SideNav extends React.Component {
     const { isISV } = user;
     const role = isISV ? 'isv' : user.role;
     const key = this.getMatchKey();
-
-    return subNavMap[role][key] || {};
+    return (_.isObject(subNavMap[role]) && subNavMap[role][key]) || {};
```
Comment (iwisunny) [inline src/components/Layout/SideNav/index.jsx]: Why not use lodash get?

File: src/routes/index.js [hunk 2]
```diff
@@ -68,11 +68,6 @@ const routes = {
   '/:dash/runtime/edit/:runtimeId': Dash.RuntimeAdd,
   '/:dash/runtime/:runtimeId': Dash.RuntimeDetail,
 
-  '/:dash/repos': Dash.Repos,
```
Comment (iwisunny) [inline src/routes/index.js]: Maybe comment it

File: src/components/Layout/SideNav/index.jsx [hunk 3]
```diff
@@ -160,6 +159,10 @@ class SideNav extends React.Component {
     const subNavData = this.getSudNavData();
     const { path } = this.props.match;
 
+    if (!subNavData.title) {
```
Comment (iwisunny) [inline src/components/Layout/SideNav/index.jsx]: does `subNavData` always an object?

File: src/components/Layout/SideNav/index.jsx (hunk 1 above)
Comment (whDongRui) [reply src/components/Layout/SideNav/index.jsx]: There role and key is variable

File: src/components/Layout/SideNav/index.jsx (hunk 3 above)
Comment (whDongRui) [reply src/components/Layout/SideNav/index.jsx]: getSudNavData method return "|| {}"

File: src/components/Layout/SideNav/index.jsx (hunk 1 above)
Comment (whDongRui) [reply src/components/Layout/SideNav/index.jsx]: ok

**Pull request merged**